docs/installation/presto-admin-installation.rst

```diff
@@ -5,19 +5,42 @@ Installing Presto Admin
 =======================
 *Prerequisites:* `Python 2.6 or Python 2.7 <https://www.python.org/downloads>`_
 
+To install ``presto-admin``:
+
+Presto Admin is packaged as an offline installer --
+``prestoadmin-<version>-offline.tar.bz2`` -- and as an online
+installer -- ``prestoadmin-<version>-online.tar.bz2``. The offline
+installer includes all of the dependencies for ``presto-admin``, so it
+can be used on a cluster without an outside network connection. The
+offline installer is currently only supported on RedHat Linux 6.x or
+CentOS equivalent.
+
+The online installer downloads all of the dependencies when you call
+``./install-prestoadmin.sh``. You need to use the online installer
+for installation of Presto on Amazon EMR.
+
+You may also need to use the online installer if you try to install
+``presto-admin`` on an operating system other than listed above,
+because some of the binary files are dependent on the system files of
+a given operating system. Be aware, though, that there may be other
+operating system dependent differences beyond the installation process
+for unsupported operating system, and ``presto-admin`` may not work.
+ 
+1. Copy the installer ``prestoadmin-<version>-offline.tar.bz2`` to the
```

Comment:
There are a lot of notes following this step in the ordered list. Suggest s/1./Before beginning the installation process, .../